media-gfx/converseen/converseen-0.9.11.1.ebuild

```diff
@@ -0,0 +1,67 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake xdg-utils
+
+DESCRIPTION="Batch image converter and resizer based on ImageMagick"
+HOMEPAGE="https://converseen.fasterland.net/
+	https://github.com/Faster3ck/Converseen/"
+SRC_URI="https://github.com/Faster3ck/Converseen/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="debug"
+
+RDEPEND="
+	dev-qt/qtcore:5
+	dev-qt/qtgui:5
+	dev-qt/qtnetwork:5
+	dev-qt/qtwidgets:5
+	|| ( media-gfx/imagemagick[cxx] media-gfx/graphicsmagick[cxx,imagemagick] )
+"
+DEPEND="${RDEPEND}"
+BDEPEND="
+	dev-qt/linguist-tools:5
+"
+
+S="${WORKDIR}/${P^}"
+
+PATCHES=(
+	"${FILESDIR}/${PN}-0.9.9.0-appdata-path.patch"
+	"${FILESDIR}/${PN}-0.9.9.0-no-update.patch"
+	"${FILESDIR}/${PN}-0.9.11.1-graphicsmagick-support.patch"
+)
+
+src_prepare() {
+	cmake_src_prepare
+
+	if has_version "media-gfx/graphicsmagick[cxx,imagemagick]"; then
```

Comment:
because of the "not ABI compatible thing", right now, this can become "stale" and encode the wrong result in the installed coverseen.

(Imagine if you install ImageMagick then install GraphicsMagick instead.)

Just change this to be "if use graphicsmagick" instead.

Reply:
True. Added it.